android/build.gradle

```diff
@@ -3,16 +3,17 @@ def safeExtGet(prop, fallback) {
 }
 
 buildscript {
-    repositories {
-        google()
-        maven {
-          url 'https://maven.google.com'
+    // The Android Gradle plugin is only required when opening the android folder stand-alone.
+    // This avoids unnecessary downloads and potential conflicts when the library is included as a
+    // module dependency in an application project.
```

Comment:
@friederbluemle this is not really the place to mention it but this just came through and reminded me - the one place I've seen this style fail is react-native-webview. It depends on kotlin, and that has to be on the classpath or the module build breaks a project build, but most projects (? my project anyway) does not by default have kotlin yet. So until (if?) the react-native template is altered to include kotlin not all modules may be de-duplicated like this. To be clear: not related to this PR, just for information

Reply:
Thanks for the heads-up @mikehardy 👌